lib/airbrake-ruby/filter_chain.rb

```diff
@@ -90,5 +91,14 @@ def root_directory_filter(root_directory)
         end
       end
     end
+
+    def thread_context_filter(thread_context)
+      proc do |notice|
+        unless (ctx = thread_context.to_h).empty?
+          notice[:params][:thread_context] = ctx
+          thread_context.clear
```

Comment:
@kyrylo so this guarantees that after a `notify` whatever that was in the `thread_context` gets cleared out?